packages/server/yarn.lock

```diff
@@ -1091,12 +1119,128 @@
     svelte-flatpickr "^3.2.3"
     svelte-portal "^1.0.0"
 
-"@budibase/pro@1.0.127":
-  version "1.0.127"
-  resolved "https://registry.yarnpkg.com/@budibase/pro/-/pro-1.0.127.tgz#a8bcffb8ccc6afde64370b3a3dc22e2d0dd04f46"
-  integrity sha512-dj0SFTmO8JuMQ97/Ik6jVPQsh9AW7U5Wkgpa4yeNfwWw3DvSoktCxpeZ9mND6BR/DWTaeZljFKsQ6uk6nimzdQ==
+"@budibase/bbui@^1.0.131":
```

Comment:
think you need to run `yarn clean && yarn bootstrap` to remove these lerna dependencies. did you run a `yarn add` or do a manual `yarn install` in server? seems that is the scenario that brings them in

Reply:
Yeah, I ran it to solve the merge conflicts. This is something we've ran into quite a bit recently - might be nice for us to automate it in some way before merging